doc/source/conf.py

```diff
@@ -107,6 +116,24 @@
     # type, unless multiple values are being returned"
 }
 
+# Manage errors
+pyvista.set_error_output_file("errors.txt")
```

Comment:
In a follow up PR you could add this file to the artifacts (for better error tracking).